index.html

```diff
@@ -16,6 +16,73 @@
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
+    <!DOCTYPE html>
+    <html lang="en">
+    
+    <head>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
+    
+      <title>Donald Fagen</title>
+      <link href="//fonts.googleapis.com/css?family=Roboto:400,500,300" rel="style" type="text/css" />
+      <link rel="stylesheet" href="style.css" />
+      <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    </head>
+    
+    
+    <!-- Add your HTML markup here -->
+    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    
+    <header>
+      <h1> Donald Fagen </h1>
+      <h2> American singer, songwriter </h2>
+      <img src='images/img1.png' alt="Donald Fagen" id="Donald Fagen - The Nightfly" </header>
+    
+      <section>
+    
+        <hr>
+    
+        <section class="introduction">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris iaculis, sem quis venenatis consequat, dui orci
+            placerat
+            erat, quis mollis tortor tortor vitae erat.
+            Proin varius gravida suscipit. Etiam erat massa, luctus eget posuere vel, porttitor sed tortor. Phasellus non
+            semper
+            libero, in faucibus mauris.
+            Vivamus cursus, nisi non tempus porta, nisi sem convallis tortor, non consequat augue dolor quis mi.Nullam
+            dapibus lorem
+            vitae ante vulputate, non ultrices mi ultrices.
+            Phasellus suscipit ex et nunc porttitor, at cursus est malesuada.</p>
+    
+          <p><img src='images\cuba_car.png' alt="City of Havana" id="Donald Fagen - The Nightfly" </p>
+    
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris iaculis, sem quis venenatis consequat, dui orci
+            placerat
+            erat, quis mollis tortor tortor vitae erat.
+            Proin varius gravida suscipit. Etiam erat massa, luctus eget posuere vel, porttitor sed tortor. Phasellus non
+            semper
+            libero, in faucibus mauris.
+            Vivamus cursus, nisi non tempus porta, nisi sem convallis tortor, non consequat augue dolor quis mi.Nullam
+            dapibus lorem
+            vitae ante vulputate, non ultrices mi ultrices.
+            Phasellus suscipit ex et nunc porttitor, at cursus est malesuada.</p>
+    
+        </section>
+    
+        </body>
+        <hr>
+    
+        <footer>
+          <img src="images/flav.png" alt="facebook logo">
+    
+          <h3><a href="https://www.facebook.com/DonaldFagenMusic"> Donald Fagen Facebook </a></h3>
+    
+    
+        </footer>
```

Comment:
This is excellent and you've made good use of the semantic tags and CSS styling. The media queries appear to work correctly when I adjust the size of my browser window.

The only suggestion I would make is you might want to make your text smaller since at the moment it's very large and makes your content take up several screens, even on a large monitor.